content/hi/docs/tasks/tls/certificate-rotation.md

```diff
@@ -0,0 +1,63 @@
+---
+reviewers:
+- divya-mohan0209
+- dipesh-rawat
+title: कुबलेट के लिए प्रमाणपत्र नियमित आवर्तन (rotation) configure करें
+content_type: task
+---
+
+<!-- overview -->
+यह पृष्ठ दिखाता है कि कुबलेट के लिए प्रमाणपत्र रोटेशन को कैसे सक्षम और configure करें।
+
+
+{{< feature-state for_k8s_version="v1.19" state="stable" >}}
+
+## {{% heading "prerequisites" %}}
+
+
+* Kubernetes संस्करण 1.8.0 या बाद का आवश्यक है।
+
+
+
+<!-- steps -->
+
+## अवलोकन
+
+कुबलेट प्रमाणपत्रों का उपयोग Kubernetes API के साथ प्रमाणित करने के लिए करता है। 
+डिफ़ॉल्ट रूप से, ये प्रमाणपत्र एक वर्ष की समाप्ति के साथ जारी किए जाते हैं ताकि उन्हें बार-बार नवीनीकृत करने की आवश्यकता न हो।
+
+Kubernetes में [कुबलेट प्रमाणपत्र
+रोटेशन](/docs/reference/access-authn-authz/kubelet-tls-bootstrapping/) शामिल है,
+जो स्वचालित रूप से एक नई कुंजी उत्पन्न करेगा और वर्तमान प्रमाणपत्र की समाप्ति के करीब आने पर Kubernetes API से एक नया प्रमाणपत्र अनुरोध करेगा। 
+एक बार नया प्रमाणपत्र उपलब्ध हो जाने पर, यह Kubernetes API के साथ कनेक्शन को प्रमाणित करने के लिए उपयोग किया जाएगा।
+
+## क्लाइंट प्रमाणपत्र रोटेशन सक्षम करना
+
+`kubelet` प्रक्रिया एक तर्क `--rotate-certificates` स्वीकार करती है जो नियंत्रित करती है कि क्या कुबलेट स्वचालित रूप से वर्तमान में उपयोग किए जा रहे प्रमाणपत्र की समाप्ति के करीब आने पर एक नया प्रमाणपत्र अनुरोध करेगा।
+
+`kube-controller-manager` प्रक्रिया एक तर्क स्वीकार करती है 
```

Comment:
Unless it's an option passed to a command or a command itself, please transliterate the components such as kubelet, kube-controller-manager etc.